lib/mongoid_session_store/store.rb

```diff
@@ -9,7 +9,7 @@ class Store < ActionDispatch::Session::AbstractStore
   private
 
     def generate_sid
-      BSON::ObjectId.new.to_s
+      SecureRandom.hex(32)
```

Comment:
@FlammenB Yeah, that's why I think we can ignore the risk. 😄  The odds of a collision happening is less than 0.000000000000000001%